src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -61,6 +61,21 @@ AuctionHouseObject* AuctionHouseMgr::GetAuctionsMap(uint32 factionTemplateId)
         return &mNeutralAuctions;
 }
 
+AuctionHouseObject* AuctionHouseMgr::GetAuctionsMapByHouseId(uint8 auctionHouseId)
+{
+    if (sWorld->getBoolConfig(CONFIG_ALLOW_TWO_SIDE_INTERACTION_AUCTION))
+        return &mNeutralAuctions;
+
+    if (!auctionHouseId)
```

Comment:
Not sure it'd be benficial. The options are CONFIG_ALLOW_TWO_SIDE_INTERACTION_AUCTION, we can't switch on that.

!auctionHouseId, well we can assume that means 0 and we could switch on that.
2 and 6
something else.

So we could have switch and check for 0, 2, 6 and default. But, would it really be worthwhile?